src/app/about/about.component.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
```

Comment:
remove all doctypes since it is already included inside the html entry file

Reply:
It's done